extensions/dbgobject/core/dbgobject.js

```diff
@@ -814,6 +814,10 @@ Loader.OnLoad(function() {
             return "???";
         }
 
+        function signedOrUnsignedVals(character, length) {
+            return (character.type.name() == "unsigned char") ? character.uvals(length) : character.vals(length);
```

Comment:
I think `vals` is supposed to do this automatically?

Reply:
Ah yes, it should. Turns out there's a bug in isUnsigned - that's why it wasn't working.